src/main/java/ch/sbb/polarion/extension/pdf/exporter/util/html/HtmlLinksHelper.java

```diff
@@ -0,0 +1,72 @@
+package ch.sbb.polarion.extension.pdf.exporter.util.html;
+
+import ch.sbb.polarion.extension.pdf.exporter.properties.PdfExporterExtensionConfiguration;
+import ch.sbb.polarion.extension.pdf.exporter.util.FileResourceProvider;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class HtmlLinksHelper {
+    private static final Pattern LINK_PATTERN = Pattern.compile("<link\\s*[^>]*>", Pattern.CASE_INSENSITIVE);
+    private static final Pattern ATTRIBUTE_PATTERN = Pattern.compile("([\\w-]+)\\s*=\\s*(['\"])(.*?)\\2");
+
+    private final Set<LinkInternalizer> linkInliners;
+
+    public HtmlLinksHelper(FileResourceProvider fileResourceProvider) {
+        this (Set.of(
+                new ExternalCssInternalizer(fileResourceProvider)
+        ));
+    }
+
+    public HtmlLinksHelper(Set<LinkInternalizer> linkInliners) {
+        this.linkInliners = linkInliners;
+    }
+
+    public String internalizeLinks(String htmlContent) {
+        boolean enablingProperty = PdfExporterExtensionConfiguration.getInstance().getInternalizeExternalCss();
```

Comment:
Isn't name like `linksInternalizationEnabled` better than `enablingProperty`?

Reply:
I thought the context is already defined by class name and method name, and wouldn't like repeat it. But if it more readable, ok to rename